HomeWork/src/main/java/HomeWork6/Employee.java

```diff
@@ -0,0 +1,57 @@
+package HomeWork6;
+
+public class Employee {
+    private String username;
```

Comment:
It's not really a `username`, to be honest. Would be better to rename this variable to `fullName`. Please rename getter and setter as well